counterexamples/transportation/segment/road/bad-road-invalid-min-occupancy.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "invalidMinOccupancy",
```

Comment:
nit: It's useful to specify in the name why is the `minOccupancy` invalid. 
e.g. 
- `bad-road-invalid-min-occupancy-type.json`
- `"id": "invalidMinOccupancyType"`

Not a blocker.